### PR TITLE
Update faq.md

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request is to add a new way to help users to reduce the scan cost.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[Add a new way to optimise the scan cost](https://sumologic.atlassian.net/browse/DOCS-348)

### CODE REVIEW

File: docs/manage/partitions/flex/faq.md [hunk 1]
```diff
@@ -48,12 +48,13 @@ Your **Account Overview** page shows the credits your org has consumed for Flex
...
 
 * Logically apportion your data, depending on how you plan to query it, into multiple partitions, instead of creating a single large partition. Try to limit queries to a single partition as much as possible. Assume you have three different services sending logs to Sumo Logic. There a couple of approaches you can take:
     * **Create one partition per service**. All the logs for that service go there. This approach can be leveraged if each team owns a service, and they typically query the logs within their service to find issues.
     * **Create one partition per log type (debug, infra, and so on)**. This approach can be leveraged if, most of the time, the team searches one log type per query. For example, the team only queries debug logs, and typically queries other logs like Database logs and OS logs in separate queries. 
-* Try to use a restrictive time range, instead of running a query with a very long time range. 
+* Try to use a restrictive time range, instead of running a query with a very long time range.
+* Using index/indexes with the metadata fields will narrow down the volume of data scanned, this in turn wil help you to reduce the scan cost.
```
Comment (JV0812) [inline docs/manage/partitions/flex/faq.md:57]:
```suggestion
* Using index/indexes with the metadata fields will narrow down the volume of data scanned, this in turn will help you to reduce the scan cost.
```

File: docs/manage/partitions/flex/faq.md [hunk 2]
```diff
@@ -48,12 +48,13 @@ Your **Account Overview** page shows the credits your org has consumed for Flex
...
 
 * Logically apportion your data, depending on how you plan to query it, into multiple partitions, instead of creating a single large partition. Try to limit queries to a single partition as much as possible. Assume you have three different services sending logs to Sumo Logic. There a couple of approaches you can take:
     * **Create one partition per service**. All the logs for that service go there. This approach can be leveraged if each team owns a service, and they typically query the logs within their service to find issues.
     * **Create one partition per log type (debug, infra, and so on)**. This approach can be leveraged if, most of the time, the team searches one log type per query. For example, the team only queries debug logs, and typically queries other logs like Database logs and OS logs in separate queries. 
-* Try to use a restrictive time range, instead of running a query with a very long time range. 
+* Try to use a restrictive time range, instead of running a query with a very long time range.
+* Using index/indexes with the metadata fields will narrow down the volume of data scanned, this in turn will help you to reduce the scan cost.
```
Comment (jpipkin1) [inline docs/manage/partitions/flex/faq.md:57]:
```suggestion
* Use indexes with the metadata fields to narrow down the volume of data scanned. This in turn will help you to reduce the scan cost.
```

**Review by jpipkin1 — APPROVED** [review]
I have only one suggestion, @JV0812.

File: docs/manage/partitions/flex/faq.md [hunk 3]
```diff
@@ -48,12 +48,13 @@ Your **Account Overview** page shows the credits your org has consumed for Flex
...
 
 * Logically apportion your data, depending on how you plan to query it, into multiple partitions, instead of creating a single large partition. Try to limit queries to a single partition as much as possible. Assume you have three different services sending logs to Sumo Logic. There a couple of approaches you can take:
     * **Create one partition per service**. All the logs for that service go there. This approach can be leveraged if each team owns a service, and they typically query the logs within their service to find issues.
     * **Create one partition per log type (debug, infra, and so on)**. This approach can be leveraged if, most of the time, the team searches one log type per query. For example, the team only queries debug logs, and typically queries other logs like Database logs and OS logs in separate queries. 
-* Try to use a restrictive time range, instead of running a query with a very long time range. 
+* Try to use a restrictive time range, instead of running a query with a very long time range.
+* Use indexes with the metadata fields to narrow down the volume of data scanned. This in turn will help you to reduce the scan cost.
```
Comment (JV0812) [inline docs/manage/partitions/flex/faq.md:57]:
```suggestion
* Use indexes with the metadata fields to narrow down the volume of data scanned. This in turn will help you to reduce the scan cost. For example, if you're querying data from the `k8s_host` source and it's being sent to multiple partitions, you can limit the scope of your search to the `abc` partition by using `_index=abc` for the selected time range.
    ```sql
    _index=abc AND _sourceCategory=k8s_host
    ```
```

**Comment by jpipkin1** [comment]
@JV0812, can we merge this PR?

**Comment by JV0812** [comment]
I was just waiting for confirmation from Manas, we can merge this now @jpipkin1.

**Pull request merged**